src/index.html

```diff
@@ -11,12 +11,101 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&amp;display=swap"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="head_a"
+        href="index.html"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo"
```

Comment:
The `alt` attribute for the image should describe the image content. 'Moyo' is not descriptive enough. Try something like 'Moyo company logo'.